python/trace-python.c

```diff
@@ -886,33 +858,32 @@ static struct uftrace_python_symbol *convert_function_addr(PyObject *frame, PyOb
 	}
 	else {
 		code = args;
-		Py_INCREF(code);
```

Comment:
This is needed to maintain the same reference count with  the`if` case.

Reply:
nice catch.